packages/renderer/src/lib/ui/TabButton.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+export let title: string;
+export let selected: boolean;
+</script>
+
+<button
+  on:click
+  class="pb-2 border-b-[3px] border-charcoal-700 whitespace-nowrap hover:cursor-pointer px-4 py-2 text-gray-600 no-underline"
+  class:border-purple-500="{selected}"
+  class:hover:border-charcoal-100="{!selected}"
+  class:text-white="{selected}"
+  >{title}
+</button>
```

Comment:
hello, I have a question

should it not be based on Podman Desktop `<Button>` or add selected flag on `<Button>` ?

Reply:
so I'm wondering if it's not another type of button as we have for example 'Link' type that displays it in a different manner

https://github.com/containers/podman-desktop/blob/71bad3bef2bc53d19792c70c3957e9f1436259e5/packages/renderer/src/lib/ui/Button.ts#L25

so if I say `<Button type="tab">` it'll be displayed as a Tab button ?

It's a suggestion, just to make sure that usage of `<button>` would be only inside Podman Desktop `<Button>` component

I'll see what others think about it

Reply:
TBH I like the GitHub Open/Closed equivalent where these would just be link Buttons, but I suppose we should follow the design. :)
We already have the Tab component, it's just hardwired to links. I guess ideally I agree there would be a Button type tab and then Tab would just use it and add link.

Reply:
hello @benoitf , any update?

Reply:
@feloy I think `<Button type="tab"` looks like the straightforward way

Reply:
I wonder if we want to add some assertion on the `selected` param, to be used only for type='tabs', of if we may want to use for other types?

Reply:
I think it may be ok to keep it simple like a new parameter but only used by tab mode